### PR TITLE
Revert "Removes sulphuric acid requirement from most computer boards"

### PR DESCRIPTION
**Pull request by praisenarsie**
Reverts tgstation/tgstation#34291

yes because making circuit boards out of glass is realistic.

as cyberboss said, "I for one enjoy the little forced interaction of the scientist and chemist, because you know it WILL happen every round"

this is just stupid powercreep by kevinz

### CODE REVIEW

**Comment by optimumtact** [comment]
Don't make a kneejerk revert of something merged by the design lead, you won't get anywhere.

in 24 hours this will be closed

**Comment by FantasticFwoosh** [comment]
Well its not like we have a code feedback subforum & ideas forum to discuss the practicality of proposed & active PR changes without forcing maintainers to make snap decisions to pull PR's for sometimes lame & self invested reasons. **_Wait_**.

File: code/modules/research/designs/comp_board_designs.dm [hunk 1]
```diff
@@ -5,6 +5,7 @@
 	desc = "I promise this doesn't give you syndicate goodies!"
 	build_type = IMPRINTER
 	materials = list(MAT_GLASS = 1000)
+	reagents_list = list("sacid" = 20)
```
Comment (kevinz000) [inline code/modules/research/designs/comp_board_designs.dm]: test

**Comment by francinum** [comment]
The amount of interaction was either so minor it hardly mattered, or just resulted in a breakin to steal 100 units of acid to make your own mini-dispenser.

**Comment by Cyberboss** [comment]
I don't like it either, but enough with the knee jerks

**Comment by lzimann** [comment]
if you want real interaction make it use different chemicals so it's not sulphuric acid for everything

**Pull request closed without merging**